.travis.yml

```diff
@@ -7,8 +7,7 @@ env:
   matrix:
     - MOLECULE_DISTRO="centos:7"
     - MOLECULE_DISTRO="centos:8"
-    - MOLECULE_DISTRO="fedora:28"
-    - MOLECULE_DISTRO="fedora:29"
+    - MOLECULE_DISTRO="fedora:31"
```

Comment:
Do you need this to test the changes? if not, I would prefer to leave it for later PR, as it is not related to the subject of this PR.